src/commands/Tools/whois.ts

```diff
@@ -63,12 +65,16 @@ export default class extends SkyraCommand {
 		const fields = message.language.get(LanguageKeys.Commands.Tools.WhoisMemberFields, { member });
 
 		const embed = new MessageEmbed()
-			.setColor(member.displayColor || BrandingColors.Secondary)
-			.addField(titles.joined, member.joinedTimestamp ? fields.joinedWithTimestamp : fields.joinedUnknown)
-			.addField(titles.createdAt, fields.createdAt)
-			.setAuthor(member.user.tag, member.user.displayAvatarURL({ size: 128, format: 'png', dynamic: true }))
-			.setDescription(member.toString())
+			.setColor(member.displayColor || Colors.White)
+			.setAuthor(
+				`${member.user.tag}${member.user.bot ? ` ${Emojis.Bot}` : ''}`,
```

Comment:
Have you even tested this? How does it look now? Do emojis render in the author field? Why is the avatar shown 3 times?